e2e/visual/components/CartesianInput.jsx

```diff
@@ -6,27 +6,39 @@ import { Cartesian } from '@compositor/kit'
 import Input from '../../../packages/Input'
 import Tooltip from '../../../packages/Tooltip'
 
-const CartesianInput = props => (
-  <React.Fragment>
+const InputNormal = props => (
```

Comment:
`InputNormal`, `InputError`, and `InputTooltip` are all identical. Can we have a single component?